site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java

```diff
@@ -3860,7 +3860,12 @@ private void addAccess(Context context, Map<String, AdditionalRole> access) {
 	 */
 	protected Map<String, AdditionalRole> loadAdditionalRoles() {
 		Map<String, AdditionalRole> additionalRoles = new HashMap<String, AdditionalRole>();
-		for (String roleId : authzGroupService.getAdditionalRoles()) {			
+		for (String roleId : authzGroupService.getAdditionalRoles()) {
```

Comment:
This looks to be the wrong place todo this check, the existing checks are done in the `AuthzGroupService` so keeping all the checks in the same place makes much more sense and the code more understandable in the future. I know the property name refers to Site Info but I think that just historic.